### PR TITLE
FALCON-1899 : Create examples artifact module in Falcon

### PR DESCRIPTION
**Pull request by peeyushb**

### CODE REVIEW

**Comment by bvellanki** [comment]
:+1:

File: examples/pom.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                <dependency>
+                    <groupId>org.apache.hadoop</groupId>
+                    <artifactId>hadoop-client</artifactId>
+                </dependency>
+            </dependencies>
+        </profile>
+    </profiles>
+
+</project>
+
```
Comment (vramachan) [inline examples/pom.xml]: extra new line.

**Comment by vramachan** [comment]
@peeyushb except one minor extra new line (test patch should catch it), everything looks ok. 
+1.

**Comment by peeyushb** [comment]
Thanks for doing the review. Removed extra line and updated the pull request.

**Comment by vrangan** [comment]
Looks good.   Don't we need documentation update about the changes.   Can you follow it up in another JIRA

**Comment by peeyushb** [comment]
@vrangan I don't think so that documentation changes are required for this pull request as it is not the user level change. User will still get examples directory under $FALCON_HOME as usual.

**Comment by vrangan** [comment]
+1.  I was meaning developer documentation - but I understand we need to create one to keep track.   An action to take up for future :)

**Comment by peeyushb** [comment]
If there are no further comments on this pull request, I will commit this soon.

**Pull request closed without merging**